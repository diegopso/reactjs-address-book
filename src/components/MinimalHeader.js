import React from 'react'

const MinimalHeader = () => {
  return (
    <header className="minimal-header"></header>
  )
}

export default MinimalHeader
