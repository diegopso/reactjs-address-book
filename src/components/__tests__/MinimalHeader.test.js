import React from 'react'
import { describe, it, expect } from '@jest/globals'
import { shallow } from 'enzyme'
import MinimalHeader from '../MinimalHeader'

describe('Testing MinimalHeader component:', () => {
  it('Renders correctly.', () => {
    const wrapper = shallow(<MinimalHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
