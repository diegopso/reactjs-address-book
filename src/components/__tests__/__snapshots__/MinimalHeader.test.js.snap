// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing MinimalHeader component: Renders correctly. 1`] = `
<header
  className="minimal-header"
/>
`;
